tlsdialer.go

```diff
@@ -188,6 +189,7 @@ func (d *Dialer) DialForTimings(network, addr string) (*ConnWithTimings, error)
 		chid = tls.HelloGolang
 	}
 	conn := tls.UClient(rawConn, configCopy, chid)
+	conn.SetSessionState(d.ClientSessionState)
```

Comment:
Looks like this still has an effect even if `d.ClientSessionState == nil`:
https://github.com/refraction-networking/utls/blob/master/u_conn.go#L100

Seems that it will just set it will just communicate that session tickets are supported.  TBH, I'm not sure what the implications are.

Reply:
Good point, will fix.